### ISSUE TITLE
Add the started Date accessible

### ISSUE THREAD

**Issue by misterfifi1** [rcarz/jira-client#132]
Hi, 

In JIRA the real date to be used for the Worklog is the started date, but this one is protected, could you add the getter? 

thanks

**Comment by Kreinoee** [comment]
This has already been fixed: #86
However there has not yet been released a version that include this fix. But if you build the project from source, you get a jar including it.

Note however that there is another problem regarding these fields: #156

**Comment by misterfifi1** [comment]
Thanks, but it is not available on maven, did you plan to release a new version?

**Comment by Kreinoee** [comment]
I cannot answer that, as I am not a developer on this project, but there is another issue requesting a new version i maven: #142

Your could try to vote on that issue.

In the mean time, you can checkout the project, run `mvn package` and then use the jar from the target dir. Your can point maven to a local jar by adding the dependency like this:

```
<dependency>
    <groupId>sample</groupId>
    <artifactId>com.sample</artifactId>
    <version>1.0</version>
    <scope>system</scope>
    <systemPath>${project.basedir}/src/main/resources/yourJar.jar</systemPath>
</dependency>
```

**Comment by rcarz** [comment]
@misterfifi1 Yes there will be a release to maven. I need to work through some of these open issues first.

**Comment by misterfifi1** [comment]
Thanks, do you know how can I be alerted when a new version is released?

**Comment by rcarz** [comment]
I'll post in #105 and here.

**Comment by MrGraversen** [comment]
Hi, it has been more than a year since this issue was reported. Is there any hope for resolution?

**Comment by vladimirsvicevic** [comment]
Hi @Kreinoee, I tried method you suggested with locally building jar, but there's some issue with testing methods, so buildding jar failed on methods: testSimpleSearch() and testExpandingChangeLogInSearch() I think. Can you try build jar or send me somehow if you succeed? 

Also, @rcarz do you know approximately when will be released new version of jira-client?

Thanks in advance.

**Comment by Kreinoee** [comment]
I have attached the jar file that I use. I cannot remember exactly what commit it is build from, but it should include fix from #86. I have gzipped it, to make github accept it, but when uncompressed, the jar files should have sha1sum  7e0e9990451fbe1650293b73b6a8e364481ea2e8

[jira-client-cirque-build-7.jar.gz](https://github.com/rcarz/jira-client/files/1455264/jira-client-cirque-build-7.jar.gz)
